src/licenses/opensource.toml

```diff
@@ -0,0 +1,68 @@
+name = "opensource"
+description = "Une description de l'innitiative Open Source."
```

Comment:
```suggestion
description = "Quelques mots sur l'open-source"
```

Reply:
No, the tag documents the OSI which is more specific than "open-source", since the concrete definition of open-source varies across organizations

Reply:
So, why the tag's name is opensource

Reply:
Because it is shorter and more explicit for users, it is also important to note the updates between OSI and FSF to get open-source and Free/Libre two different things even if they are coupled

Reply:
I agree